package.json

```diff
@@ -102,7 +102,9 @@
     "prepublishOnly": "npm run test",
     "lint": "tslint -c ./config/tslint/tslint.json --project ./tsconfig.json",
     "build": "tsc && npm run lint",
```

Comment:
I would prefer to run `npm run build` instead of `npm run build:dev` and let travis run `npm run build:prod`.